docs/EN_US/HPCCSystemAdmin/SA-Mods/DAFtslave.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="SA_SupportUtilities">
+  <title>Support Utilities</title>
+
+  <para>There are a few additional components which are neither system servers
+  or client interfaces but nonetheless important in supporting HPCC
+  tasks.</para>
+
+  <sect3 id="DAFilesrv">
+    <title>dafilesrv</title>
+
+    <para>The dafilesrv is a daemon that runs on system servers or physical
+    locations on the HPCC system. This daemon enables the HPCC components to
+    have file access to physical file locations. File access could be
+    spraying, despraying, copying, etc. It is also used to enable file
+    security.</para>
+
+    <para>The dafilesrv daemon runs as a process on every Thor or Roxie node.
+    The daemon will typically keep running even if your system is stopped.
+    That is an important fact to keep in mind if you are stopping your system
+    for maintenance. Particularly for installations or upgrades. Even if you
+    issue the HPCC system stop command, you would still need to stop the
+    dafilesrv daemon. Issue the following command to stop the dafilesrv
+    daemon. Then verify that dafilesrv is not running before running the
+    installation package. </para>
+
+    <programlisting>sudo /etc/HPCCSystems/sbin/hpcc-run.sh -a dafilesrv stop</programlisting>
+
+    <para>You must have sudo permission level access in order to start or stop
+    any of the HPCC components, including dafilesrv.</para>
+  </sect3>
+
+  <sect3>
+    <title>ftslave</title>
+
+    <para>The ftslave is a support utility used by Thor slaves and eclagent
+    when deploying data to the nodes. The ftslave is launched on demand as
```

Comment:
It is not used by Thor or eclagent - it is only a process used by spraying by Dfu server.